app.py

```diff
@@ -38,6 +38,15 @@
     'Street Sword: Practical Use of the Long Blade for Self-Defense', 
 ]
 
+BULLSHIT_SOURCES = [
+    'not source-based',
+    'some bullshit that Shadiversity made up and called HEMA.',
+    'some HEMAism that people pass around as common knowledge even though it\'s not supported by any sources.',
+    'some weird play from Talhoffer that doesn\'t make any damn sense.',
+    'something that people made up and said it\'s from Liechtenauer, even though it isn\'t.',
+    "something Brent made up."
```

Comment:
Can we add the trailing comma and maintain quote consistency please? 
```suggestion
    'something Brent made up.', 
```

Reply:
Done